packages/engine-server/src/drivers/sqlite/SqliteMetadataStore.ts

```diff
@@ -30,6 +39,94 @@ import { SchemaNotesTableUtils } from "./tables/SchemaNotesTableUtils";
 export class SqliteMetadataStore implements IDataStore<string, NotePropsMeta> {
   constructor(private _db: Database, private _vaults: DVault[]) {}
 
+  /**
+   * Goes through all domains and recursively apply schemas.
+   */
+  async initSchema(fileStore: IFileStore, wsRoot: string, logger: DLogger) {
+    // this whole thing to parseAllNoteFilesForSqlite / SqliteDbFactory
```

Comment:
i think the beginning "// TODO: move" part of the comment got truncated.

Reply:
right. I'll fix that.